app/models/profile.rb

```diff
@@ -14,4 +14,25 @@ class Profile < ActiveRecord::Base
   validates :user, presence: true
   validates_presence_of :first_name, :last_name, :gender, :birth_date, :email, :school, :street_name, :zip_code, :city, :state, :country
 
+  def age
+    now = Date.today
+    now.year - birth_date.year - (birth_date.to_date.change(:year => now.year) > now ? 1 : 0)
+  end
+
+  def name
+    first_name + " " +  last_name
+  end
+
+  def self.human_attribute_name(*args)
```

Comment:
Isn't that exactly the type of hardcoding internationalism that we wanted to avoid?